e2e/tests/gatekeepingOrderJourney_test.js

```diff
@@ -0,0 +1,57 @@
+const config = require('../config.js');
+const directions = require('../fixtures/directions.js');
+const gatekeepingCaseData = require('../fixtures/caseData/gatekeepingNoAllocatedJudge.json');
+
+let caseId;
+
+Feature('Gatekeeping judge SDO journey');
+
+BeforeSuite(async ({I}) => {
+  caseId = await I.submitNewCaseWithData(gatekeepingCaseData);
+  await I.navigateToCaseDetailsAs(config.judicaryUser, caseId);
+});
+
+Before(async ({I}) => await I.navigateToCaseDetails(caseId));
+
+Scenario('Gatekeeping judge adds allocated judge', async ({I, caseViewPage, allocatedJudgeEventPage}) => {
+  await caseViewPage.goToNewActions(config.applicationActions.allocatedJudge);
+  await allocatedJudgeEventPage.enterAllocatedJudge('Moley', 'moley@example.com');
+  await I.completeEvent('Save and continue');
+  I.seeEventSubmissionConfirmation(config.applicationActions.allocatedJudge);
+  caseViewPage.selectTab(caseViewPage.tabs.casePeople);
+  I.seeInTab(['Allocated Judge', 'Judge or magistrate\'s title'], 'Her Honour Judge');
+  I.seeInTab(['Allocated Judge', 'Last name'], 'Moley');
+  I.seeInTab(['Allocated Judge', 'Email Address'], 'moley@example.com');
+});
+
+Scenario('Gatekeeping judge drafts gatekeeping order', async ({I, caseViewPage, addGatekeepingOrderEventPage}) => {
```

Comment:
maybe do this before adding allocated judge to verify next steps label, then add allocated judge + seal?

Reply:
e2e flow is now draft -> add allocated judge -> seal